bin/build_libmonero_java.sh

```diff
@@ -2,19 +2,25 @@
 
 #EMCC_DEBUG=1
 
-HOST_NCORES=$(nproc 2>/dev/null || shell nproc 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null || echo 1)
+export HOST_NCORES=${HOST_NCORES-$(nproc 2>/dev/null || shell nproc 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null || echo 1)}
+BUILD_DIR="$(pwd)/build"
+export CMAKE_PREFIX_PATH="$(pwd)/external/monero-cpp/build/install"${CMAKE_PREFIX_PATH+:$CMAKE_PREFIX_PATH}
+export USE_DEVICE_TREZOR=OFF
```

Comment:
Can we remove this or does it cause a problem on your system?

Reply:
if removed, trezor deps would become dynamically linked an break the concept of "library wrapper". The problem begun with protobuf static -fPIC building. Follow the historic:
- https://github.com/woodser/monero-java/pull/105#discussion_r1672333295
- https://github.com/woodser/monero-java/pull/105#issuecomment-2334244486
- https://github.com/woodser/monero-java/pull/105#issuecomment-2336690872
- https://github.com/woodser/monero-java/pull/105#issuecomment-2358270139